angular.json

```diff
@@ -158,5 +158,8 @@
     "@schematics/angular:component": {
       "style": "scss"
     }
+  },
+  "cli": {
+    "analytics": "72a6ca29-562e-4020-abd5-0c7dd3541311"
```

Comment:
What is this? Was it automatically added? 🤔

Reply:
Yeah It must have been. Should I remove it?

Reply:
Yes, I'll need to review if it's needed and what it's doing.